cmd/podman/shared/create.go

```diff
@@ -117,6 +117,28 @@ func CreateContainer(ctx context.Context, c *GenericCLIResults, runtime *libpod.
 					if err != nil {
 						return nil, nil, errors.Wrapf(err, "unable to get healthcheck for %s", c.InputArgs[0])
 					}
+
+					if healthCheck != nil {
+						hcCommand := healthCheck.Test
```

Comment:
Could hcCommand[0] == ""?

Reply:
It seems Docker build fails for `HEALTHCHECK CMD` or `HEALTHCHECK CMD []`. But `HEALTHCHECK CMD [""]` is accepted.

Reply:
Hmm, but that is mapped to `["CMD", ""]` hence the healthcheck execution will fail. I think we should check if the resulting command string is empty or only contains white space.

`["CMD-SHELL", ""]` (which Docker won't generate) leads to `/bin/sh -c ""` which of course doesn't do anything sensible, but at least doesn't fail.